src/fragments/lib/auth/js/enable-signin-signup-signout.mdx

```diff
@@ -1,8 +1,18 @@
-In this guide you will set up sign-up, sign-in, and sign-out and then test this functionality. This includes a review of sign-up methods provided by Amplify, the specific user attributes used by Amazon Cognito, how to confirm users after initial sign-up, and the differences between local and global sign-out.
+> *This guide provides essential information to help you select and complete the activities you need for your Amplify project. We organized each section around a distinct job or decision point to help you understand your available options, steps to complete, and recommended best practices.*
 
-Before you begin, you should have finished [*Set up and connected backend resources with a configured Auth category*](/lib/auth/set_up_auth#set-up-and-connect-backend-resources).
+In this guide you will set up sign-up, sign-in, and sign-out using the Amplify Libraries and then test this functionality. This includes a review of sign-up methods provided by Amplify, the specific user attributes used by Amazon Cognito, how to confirm users after initial sign-up, and the differences between local and global sign-out. Once set up you can connect it to your frontend UI. 
 
-For many apps, user sign-up and sign-in with a username/password is all that is required. Once authenticated, the app can talk to an API to access and mutate data. We will use the username/password authentication flow to get started. Both [third party social provider federation]( https://docs.amplify.aws/lib/auth/social/q/platform/js/) and [Multi-factor authentication]( https://docs.amplify.aws/lib/auth/mfa/q/platform/js/) can be added after this initial set up.
+<Callout>
+
+This guide is for those who want to set up Amplify Auth with the Amplify Libraries. If you want to create a sign in and registration experience for your app with a few lines of code we recommend using the [Authenticator component](https://ui.docs.amplify.aws/react/connected-components/authenticator) which provides a customizable UI and complete authentication flows.
+
+</Callout>
+
+Before you begin, you should have finished [*'Set up and connect backend resources'*](/lib/auth/set_up_auth#set-up-and-connect-backend-resources) with a configured Auth category.
```

Comment:
Before you begin, you should have finished the [*Set up and connect backend resources*](/lib/auth/set_up_auth#set-up-and-connect-backend-resources) guide.

Reply:
Corrected